msteams-platform/concepts/device-capabilities/device-capabilities-overview.md

```diff
@@ -37,5 +37,3 @@ After getting access to device capabilities, use Teams media capability APIs to
 * Record audio through microphone.
 * Share location using [location picker](location-capability.md).
 
-Also, you can integrate the Teams native [people picker control](people-picker-capability.md) that allows users to search and select people in the web app experience.
```

Comment:
Can we instead rephrase it to highlight this capability and not remove it completely?
To my understanding the comment was around ToC only, so we need not remove the cross referencing part within the document itself.